spring-boot-project/spring-boot-dependencies/pom.xml

```diff
@@ -51,6 +51,7 @@
 		<commons-lang3.version>3.9</commons-lang3.version>
 		<commons-pool.version>1.6</commons-pool.version>
 		<commons-pool2.version>2.6.2</commons-pool2.version>
+		<coroutines.version>1.3.0-RC</coroutines.version>
```

Comment:
I am not sure it should be called like that. I'd prefer if there was something that colocates it with Kotlin (i.e. `kotlin-coroutines.version`)

Reply:
Yeah I am open to both.